// Copyright 2012 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// Sync protocol datatype extension for an app setting.
// This is the same as for an extension setting, but uses a separate datatype
// in order to control syncability separately.

// If you change or add any fields in this file, update proto_visitors.h and
// potentially proto_enum_conversions.{h, cc}.

syntax = "proto2";

option java_multiple_files = true;
option java_package = "org.chromium.components.sync.protocol";

option optimize_for = LITE_RUNTIME;

package sync_pb;

import "components/sync/protocol/extension_setting_specifics.proto";

// Properties of app setting sync objects; just an extension setting.
message AppSettingSpecifics {
  optional ExtensionSettingSpecifics extension_setting = 1;
}
